handler/randomizer_test.go

```diff
@@ -0,0 +1,80 @@
+package handler
+
+import (
+	"github.com/stretchr/testify/assert"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+func Test_randomizeSchedule_VerifyCronSyntax(t *testing.T) {
+	formatString := "%d is not between %d and %d"
+	arr := []int{0, 0, 0, 0, 0}
+	t.Run("YearlySyntax", func(t *testing.T) {
+		for i := 0; i < 200; i++ {
+			seed := "namespace/name-" + strconv.Itoa(i) + "@backup"
+			schedule, err := randomizeSchedule(seed, "@yearly-random")
+			assert.NoError(t, err)
+			fields := strings.Split(schedule, " ")
+			for j, f := range fields {
+				number, _ := strconv.Atoi(f)
+				arr[j] = number
+			}
+			assert.InDelta(t, 0, arr[0], 59.0, formatString, arr[0], 0, 59)
+			assert.InDelta(t, 0, arr[1], 59.0, formatString, arr[1], 0, 59)
+			assert.InDelta(t, 1, arr[2], 26.0, formatString, arr[2], 1, 27)
```

Comment:
Confusion of including or not. The above tests handles the upper limit including while this does not. The lover limit is including in all assertions.

Reply:
This has to do with the cron syntax and the date.
For the minutes, we have a range between 0 and 59 of possible values (think max value = 0 + 59)
For the hours, a range between 0 and 23
However, since dates start at 1, we have different delta:
For dayOfMonth:  start at 1 + delta of 26 (max value = 1 + 26 = 27)

Or do you mean something else?

Reply:
Ah, this is what I was missing.